src/Integration/Aws/DynamoDbSession/src/SessionHandler.php

```diff
@@ -66,6 +76,11 @@ public function __construct(DynamoDbClient $client, array $options)
         $options['hash_key'] = $options['hash_key'] ?? 'id';
         $options['session_lifetime_attribute'] = $options['session_lifetime_attribute'] ?? 'expires';
         $options['id_separator'] = $options['id_separator'] ?? '_';
+        $options['consistent_read'] = $options['consistent_read'] ?? true;
+        $options['locking'] = (bool) ($options['locking'] ?? false);
```

Comment:
I think the cast to bool (same for float/int below) is not necessary

The type is already defined in the doc bloc and should be respected.

Reply:
It's only respected if the project where you're using this is using static analyzer. But ok :heavy_check_mark: